rero_ils/modules/acq_accounts/api.py

```diff
@@ -61,49 +64,333 @@ class AcqAccount(IlsRecord):
     fetcher = acq_account_id_fetcher
     provider = AcqAccountProvider
     model_cls = AcqAccountMetadata
+
     pids_exist_check = {
         'required': {
             'lib': 'library',
             'budg': 'budget'
         },
         'not_required': {
+            'parent': 'acq_account',
             'org': 'organisation'
         }
     }
 
+    _extensions = [ParentAccountDistributionCheck()]
+
+    def __hash__(self):
+        """Builtin function to return an hash for this account."""
+        # TODO :: find a better hash method
+        return hash(self.pid)
+
     @property
     def library_pid(self):
         """Shortcut for acq account library pid."""
         return extracted_data_from_ref(self.get('library'))
 
-    def get_number_of_acq_order_lines(self):
-        """Get number of acquisition order lines linked to this account."""
-        results = AcqOrderLinesSearch().filter(
-            'term', acq_account__pid=self.pid).source().count()
-        return results
+    @property
+    def library(self):
+        """Shortcut to get related library."""
+        return extracted_data_from_ref(self.get('library'), data='record')
+
+    @property
+    def organisation_pid(self):
+        """Shortcut for acq account organisation pid."""
```

Comment:
```suggestion
        """Shortcut for acquisition account organisation PID."""
```
I know, there's a lot of `pid` in docstrings. `PID` is better, but do as you can.